src/components/Details/VmDetails/VmDetails.js

```diff
@@ -0,0 +1,252 @@
+  const pod = getPod(vm, resources, VIRT_LAUNCHER_POD_PREFIX);
+
+  // TODO Fix FieldLevelHelp content for Status
+  return (
+    <Fragment>
+      <div className="co-m-pane__body">
+        <h1 className="co-m-pane__heading">Overview</h1>
+        <div className="co-m-pane__body">
+          <div className="row">
+            {/* Description column */}
+            <div className="col-sm-4">
+              <div>
+                <VmStatusReporter vm={vm} />
+              </div>
+              <dd>{get(vm, 'metadata.annotations.description', DASHES)}</dd>
+            </div>
+
+            {/* Details columns */}
+            <div className="col-sm-8">
+              <div className="row">
+                {/* Details column 1 */}
+                <div className="col-sm-4">
+                  <dl>
+                    <dt>
+                      Status
+                      <FieldLevelHelp content={"This is the VM's status"} />
+                    </dt>
+                    <dd>
+                      <StateColumn vm={vm} loaded={props.loaded} resources={props.resources} />
+                    </dd>
+
+                    <dt>Operating System</dt>
+                    <dd>{get(vm, 'metadata.annotations["os.template.cnv.io"]', DASHES)}</dd>
+
+                    <dt>IP Addresses</dt>
+                    <dd>IP Address List</dd>
+
+                    <dt>Workload Profile</dt>
+                    <dd>{get(vm, 'metadata.annotations["workload.template.cnv.io"]', DASHES)}</dd>
+
+                    <dt>Template</dt>
+                    <dd>{get(vm, 'metadata.annotations["template.cnv.ui"]', DASHES)}</dd>
+                  </dl>
+                </div>
+
+                {/* Details column 2 */}
+                <div className="col-sm-4">
+                  <dt>FQDN</dt>
+                  <dd>{pod ? pod.spec.hostname : DASHES}</dd>
+
+                  <dt>Project</dt>
+                  <dd>Project Information</dd>
+
+                  <dt>Pod</dt>
+                  <dd>
+                    <FirehoseResourceLink
+                      loaded={loaded}
+                      resources={resources}
+                      kind={PodModel.kind}
+                      filter={data => findPod(data, vm.metadata.name)}
+                      ResourceLink={ResourceLink}
+                    />
+                  </dd>
+                </div>
+
+                {/* Details column 3 */}
+                <div className="col-sm-4">
+                  <dt>Cloud Init</dt>
+                  <dd>
+                    <OnOffReporter on={cloudInitInUse(vm)} />
+                  </dd>
+
+                  <dt>Node</dt>
+                  <dd>
+                    <NodeResourceLink loaded={loaded} vm={vm} resources={resources} NodeLink={NodeLink} />
+                  </dd>
+
+                  <dt>Flavor</dt>
+                  <dd>{get(vm, 'metadata.annotations["flavor.template.cnv.io"]', DASHES)}</dd>
```

Comment:
In case of Custom flavor I guess that we will also want to show CPU/Memory values. You can use https://github.com/kubevirt/web-ui-components/blob/master/src/utils/selectors.js#L24 and https://github.com/kubevirt/web-ui-components/blob/master/src/utils/selectors.js#L25

Reply:
Done!